docs/source/reference/special.rst

```diff
@@ -1,8 +1,8 @@
 Special Functions
 =================
 
-Faster versions of common Bessel Functions
-------------------------------------------
+Faster versions of common Bessel Functions and other special functions.
```

Comment:
It is better to make a new section for gamma-related functions. See [the SciPy documentation](https://docs.scipy.org/doc/scipy/reference/special.html): gamma functions are put to "Gamma and Related Functions", and zeta functions are put to "Other Special Functions".